### PR TITLE
Refactor about page layout.

### PR DESCRIPTION
**Pull request by MiguelMedeiros**

### CODE REVIEW

**Review by softsimon — CHANGES_REQUESTED** [review]
I think the margins are a little bit too big.

**Comment by wiz** [comment]
Sorry, what is the point of this PR? Seems to be breaking many things and margins etc. for no apparent benefit?

File: frontend/src/app/components/about/about.component.html [hunk 1]
```diff
@@ -1,290 +1,204 @@
...
-        </a>
-      </ng-template>
-    </ng-container>
-    <br><br>
-    
+    <div class="wrapper">
+      <ng-container *ngIf="sponsors$ | async as sponsors; else loadingSponsors">
+        <ng-template ngFor let-sponsor [ngForOf]="sponsors">
+          <a [href]="'https://twitter.com/' + sponsor.handle" target="_blank" rel="sponsored" [title]="sponsor.handle">
+            <img class="image" [src]="sponsor.imageUrl" />
```
Comment (wiz) [inline frontend/src/app/components/about/about.component.html:55]: This is not correct and introduces a bug

Comment (MiguelMedeiros) [reply frontend/src/app/components/about/about.component.html]: reverted:
``
to:
``

**Pull request merged**